bp_top/test/tb/bp_tethered/Makefile.verilator

```diff
@@ -41,6 +41,8 @@ $(SIM_DIR)/cce_ucode.mem: $(CCE_MEM_PATH)/$(CCE_MEM)
 	@cp $^ $@
 
 NBF_INPUTS = --ncpus=$(NCPUS) --ucode=cce_ucode.mem
+NBF_INPUTS += --domains=$(DOMAINS_P)
```

Comment:
same comment as above, probably unnecessary

Reply:
Requested for clarification above.